lib/script_helper.rb

```diff
@@ -0,0 +1,9 @@
+require 'yaml'
+
+module ScriptHelper
+  def self.read_config(binding)
+    YAML.load_file(__dir__+'/../scripts/config.yml').each do |key, value|
+      binding.local_variable_set(key.to_sym, value)
+    end
+  end
+end
```

Comment:
@afred: Good calls on both points. 
- On the `binding` thing, I had started off with the local variables, and when I needed to add the default machinery I was curious if code outside the current scope could tweak the binding... but just because you can, doesn't mean it's a good idea.
- On the other point, not secrets per se, but details that will differ between accounts. Also not necessarily required, everything could be supplied on the command line, but more clear if there was a message.
Thanks.